hazelcast/src/test/java/com/hazelcast/config/ConfigXmlGeneratorTest.java

```diff
@@ -358,11 +358,16 @@ public void testManagementCenterConfigGenerator() {
     @Test
     @SuppressWarnings("deprecation")
     public void testReplicatedMapConfigGenerator() {
+        MergePolicyConfig mergePolicyConfig = new MergePolicyConfig()
```

Comment:
I would expect that if additional information is added to the config, that additional assert statements are added.

Reply:
Hm, comparing the whole config instance was missing. Thanks for catching.

Reply:
Hi @blazember I'm still failing to see additional assertions.

I would expect to see a bunch of such statements:
```
assertEquals(NATIVE, replicatedMapConfig.getInMemoryFormat())
```
Am I missing something?

Reply:
@pveentjer Now added the missing field level assertions, plus a full config object assertion.